module3/src/controllers/user.controller.ts

```diff
@@ -0,0 +1,84 @@
+import express, { Router } from 'express';
+import { Error } from 'sequelize';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import UserService from '../services/user.service.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { validateSchema } from '../utils/utils.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { userSchema } from '../validators/user.ts';
+
+const userService = UserService();
+
+const usersRouter = Router();
+usersRouter.use(express.json());
+const userRouter = Router();
+userRouter.use(express.json());
+
+export default (app: Router) => {
+    app.use('/users', usersRouter);
```

Comment:
Move it to main file. It is help to understand the routes mapping

Reply:
Done